packages/desktop_webview_window/lib/src/webview_impl.dart

```diff
@@ -222,13 +224,13 @@ class WebviewImpl extends Webview {
   }
 
   @override
-  void addOnUrlRequestCallback(OnUrlRequestCallback callback) {
```

Comment:
the `callback`parameter should be nullable

Reply:
Done